qa/standalone/scrub/osd-scrub-repair.sh

```diff
@@ -384,12 +384,12 @@ function unfound_erasure_coded() {
     #
     # it may take a bit to appear due to mon/mgr asynchrony
     for f in `seq 1 60`; do
-	ceph -s | grep "1/1 objects unfound" && break
+	ceph -s | grep "1/2 objects unfound" && break
```

Comment:
@dzafman is hunting the root cause of this issue (http://tracker.ceph.com/issues/38631), better have him double check this..